README.md

```diff
@@ -75,11 +75,13 @@ response = await llm.generate(prompt)
 print(f"Generated dog name: {response}")
 ```
 
-<!--
-TODO:
-Add links to quickstart guides for individual packages, demonstrating how to extend this with their functionality.
-Add a link to the full tutorial.
--->
+## How Ragbits documentation is organized
+
+- [Quickstart](quickstart/quickstart1_prompts/) - Get started with Ragbits in a few minutes
```

Comment:
This won't work (will result in links pointing to GitHub's 404 page).

When I wrote about making the links relative I specifically meant internal links within the documentation, not `README.md`. I would probably revert this file to what it was before.

To make this work as a relative link within Github, you would need to make a relative link that starts from the folder when `README.md` is located, and point to specific files. For example `[Quickstart](docs/quickstart/quickstart1_prompts.md)`. But again - I think its better to point to the separate documentation site, so I would revert this one to the version that you had before.